mscore/musescore.cpp

```diff
@@ -7823,15 +7833,18 @@ void MuseScore::init(QStringList& argv)
             qApp->processEvents();
             }
 
-      if (!MScore::noGui)
+      showSplashMessage(sc, "Updating interface style and theme…");
```

Comment:
Are you sure that the string will be translated?

Reply:
Well, I called `QObject::tr()` in `showSplashMessage`, is that not enough?

Reply:
I am not sure. I think transtable strings are generated at compile time, not at runtime

Reply:
I think you're right. I'll change it to `tr()` version.

Reply:
> I am not sure. I think transtable strings are generated at compile time, not at runtime

Generated at `lupdate` time (and from the the sources), inserted/replaced at runtime